tartiflette/execution/factory.py

```diff
@@ -0,0 +1,385 @@
+        super().__init__(schema, error_coercer, cache_decorator, parser, rules)
+        self._directived_executor = schema.bake_executor(self._executor)
+
+    async def _executor(
+        self,
+        schema: "GraphQLSchema",
+        document: "DocumentNode",
+        request_parsing_errors: Optional[List["TartifletteError"]] = None,
+        operation_name: Optional[str] = None,
+        context: Optional[Any] = None,
+        variables: Optional[Dict[str, Any]] = None,
+        initial_value: Optional[Any] = None,
+    ):
+        """
+        TODO:
+        :param schema: TODO:
+        :param document: TODO:
+        :param request_parsing_errors: TODO:
+        :param operation_name: TODO:
+        :param context: TODO:
+        :param variables: TODO:
+        :param initial_value: TODO:
+        :type schema: TODO:
+        :type document: TODO:
+        :type request_parsing_errors: TODO:
+        :type operation_name: TODO:
+        :type context: TODO:
+        :type variables: TODO:
+        :type initial_value: TODO:
+        :return: TODO:
+        :rtype: TODO:
+        """
+        if request_parsing_errors:
+            return await self._response_builder(errors=request_parsing_errors)
+        return await execute(
+            schema,
+            document,
+            self._response_builder,
+            initial_value,
+            context,
+            variables,
+            operation_name,
+        )
+
+    async def __call__(
+        self,
+        query: Union[str, bytes],
+        operation_name: Optional[str] = None,
+        context: Optional[Any] = None,
+        variables: Optional[Dict[str, Any]] = None,
+        initial_value: Optional[Any] = None,
+    ) -> Dict[str, Any]:
+        """
+        Parses and executes a GraphQL query/mutation request.
+        :param query: the GraphQL request / query as UTF8-encoded string
+        :param operation_name: the operation name to execute
+        :param context: value that can contain everything you need and that
+        will be accessible from the resolvers
+        :param variables: the variables provided in the GraphQL request
+        :param initial_value: an initial value corresponding to the root type
+        being executed
+        :type query: Union[str, bytes]
+        :type operation_name: Optional[str]
+        :type context: Optional[Any]
+        :type variables: Optional[Dict[str, Any]]
+        :type initial_value: Optional[Any]
+        :return: computed response corresponding to the request
+        :rtype: Dict[str, Any]
+        """
+        document, errors = self._parse_and_validate_query(query, self._schema)
+        return await self._directived_executor(
+            self._schema,
+            document,
+            errors,
+            operation_name,
+            context,
+            variables,
+            initial_value,
+            context_coercer=context,
+        )
+
+
+class OperationSubscribor(BaseOperationPerformer):
```

Comment:
`Subscriptor` or `Subscriber` :p

Reply:
I renamed all occurrences to `Subscriptor` 😆